src/views/virtualmachines/actions/components/CloneVMModal/components/CloningStatus.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC, useEffect } from 'react';
+
+import { modelToGroupVersionKind } from '@kubevirt-ui/kubevirt-api/console';
+import VirtualMachineCloneModel from '@kubevirt-ui/kubevirt-api/console/models/VirtualMachineCloneModel';
+import { V1alpha1VirtualMachineClone } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Alert,
+  AlertVariant,
+  ProgressStep,
+  ProgressStepper,
+  ProgressStepVariant,
+  Stack,
+} from '@patternfly/react-core';
+
+import { CLONING_STATUSES, STATUS_TO_PROGRESS_VARIANT } from '../utils/constants';
+
+type CloningStatusProps = {
+  onComplete: (status: CLONING_STATUSES) => void;
+  vmCloneRequest: V1alpha1VirtualMachineClone;
+};
+
+const CloningStatus: FC<CloningStatusProps> = ({ onComplete, vmCloneRequest }) => {
```

Comment:
why onclomple is as prop if no other component is using it and can be declared here?

Reply:
`CloneVMModal` use this prop to redirect the user to the right page on succeeded and change the modal state.